packages/binding-mbus/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@node-wot/binding-mbus",
+  "version": "0.7.7",
```

Comment:
Can you update to
`"version": "0.7.8"`

Reply:
Note: I think this causes the tests to fail, see https://github.com/eclipse/thingweb.node-wot/pull/460/checks?check_run_id=3281511482
`npm ERR! 404  '@node-wot/binding-mbus@0.7.8' is not in the npm registry.`